src/components/Footer/Footer.module.css

```diff
@@ -46,4 +49,149 @@
     cursor: pointer;
     font-size: 1rem;
 
-  }
\ No newline at end of file
+  }
+
+
+  /* SubFooter styles */
+
+
+  .footerInfo {
+    background-color: #012F34;
+    display: flex;
+    justify-content: space-between;
+   padding: 2rem;
+   position: relative;
+   
+  }
+
+  .logoDesc img {
+    text-align: justify;
+    
+  }
+
+  .logoDesc ul {
+    margin-right: 3rem;
+  }
+
+  
+
+  .logoText {
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   gap: 2rem;
+   text-align: justify;
+   color: #FFFF;
+   padding-top: 1rem;
+   margin-left: 0.5rem;
+  }
+
+  .location ul {
+    display: grid;
+    text-align: justify;
+    gap: .4rem;
+    color: #979797;
+    font-size: 15px;
+  }
+
+  .footerInfo ul li {
+   list-style: none;
+  }
+
+  .socialLinks p {
+    color: #979797;
+    font-size: 15px;
+    padding-top: 1rem;
+    text-align: justify;
+
+
+
+  }
+
+  .socialLinks ul {
+    display: flex;
+    gap: 0.8rem;
+
+  }
+
+
+  /* Media Queries */
+
+/* Small devices (landscape phones, 576px and up) */
```

Comment:
Please comply with the breakpoints we provided in the slack channel with

Reply:
Okay thank you I will make the necessary changes